react/react1/week1/src/App.js

```diff
@@ -0,0 +1,34 @@
+import React, {useState} from 'react';
+import ToDoList from './ToDoList'
+
+function App() {
+  const [todos, setTodos] = useState([
```

Comment:
No need to use the **useState hook** here, as you are giving the fix values to the todos array, you can use useState hook in the case when your vaues are dynamic (when the state changes) i.e when the values changes. see more about this [here](https://reactjs.org/docs/hooks-state.html)